scripts/jest/root.config.js

```diff
@@ -27,9 +27,9 @@ module.exports = {
     ],
     coverageThreshold: {
         global: {
-            branches: 80,
-            functions: 90,
-            lines: 90,
```

Comment:
We should probably get more fine-grained about these at some point, or figure out how to merge it with our Karma coverage.